docs/raw/9-blockchain-interactions.md

```diff
@@ -0,0 +1,186 @@
+---
+permalink: /spec/9
+parent: Raw specs
+title: 9/BLOCKCHAIN-USAGE
+---
+
+# 3/BLOCKCHAIN-USAGE
+
+> Version: 0.1
+>
+> Status: Raw
```

Comment:
How come this is marked as Raw? At what point do you imagine this would be Draft or Stable? My understanding is that this describes how we currently interact with the blockchain.